proto/commons/massa/model/v1/draw.proto

```diff
@@ -17,10 +17,10 @@ option swift_prefix = "MMODEL";
 
 // Slot draw
 message SlotDraw {
-  // Block producer address
-  string block_producer = 1;
   // Slot
-  massa.model.v1.Slot slot = 2;
+  massa.model.v1.Slot slot = 1;
+  // optional Block producer address
+  google.protobuf.StringValue block_producer = 2;
```

Comment:
add "optional" in front to make it explicit that it is optional

Reply:
done via StringValue